src/main/java/org/launchcode/exercisemeetup/Controllers/ActivityController.java

```diff
@@ -78,11 +78,15 @@ public String processAddActivity(@ModelAttribute @Valid Activity newActivity, Er
 
     }
 
-    @RequestMapping(value="view/{activityId}",method = RequestMethod.GET)
-    public String viewActivity(@PathVariable int activityId, Model model, HttpSession httpSession){
+    @RequestMapping(value="view/{activityId}")
+    public String viewActivity(@PathVariable int activityId, Model model, HttpSession httpSession, @RequestParam(value="completed", required = false) boolean completed){
 
         Optional<Activity> activity = activityDao.findById(activityId);
         Activity activityInfo = activity.get();
+        if(completed == true) {
```

Comment:
@branperez  this can be simplified to `if (completed) {`

Reply:
Ohhhh, you're right. I'll push that with the user profile if that's kewl